### PR TITLE
Migrate RocketChat to use RHCC

### PR DESCRIPTION
**Pull request by sabre1041**
**DO NOT MERGE** - [Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=1462606)

Migrate source of RocketChat to the Red Hat Container Catalog

### CODE REVIEW

**Comment by sabre1041** [comment]
Until the referenced bug is fixed, the `image` field of the _DeploymentConfig_ can be statically set to `registry.connect.redhat.com/rocketchat/rocketchat:0.56.0`

**Comment by sabre1041** [comment]
@etsauer I've updated the PR to use direct reference to the image instead of using ImageStreams.

Please review and merge if applicable

**Comment by oybed** [comment]
LGTM

**Review by oybed — APPROVED** [review]

**Comment by oybed** [comment]
@etsauer changes look good to me - please review and merge if you agree

**Comment by sabre1041** [comment]
@oybed FYI i made a minor doc fix after you approved the PR.

**Comment by oybed** [comment]
@sabre1041 still LGTM

File: mongodb-statefulset-replication.yaml [hunk 1]
```diff
@@ -191,7 +191,6 @@ objects:
               # Uncomment this if using dynamic volume provisioning.
               # https://docs.openshift.org/latest/install_config/persistent_storage/dynamically_provisioning_pvs.html
               # volume.alpha.kubernetes.io/storage-class: anything
-              volume.alpha.kubernetes.io/storage-class: dynamic
```
Comment (etsauer) [inline mongodb-statefulset-replication.yaml:194]: @sabre1041 why are we removing this? i don't see anything in the template or documentation that creates the PV for me.

**Review by etsauer — CHANGES_REQUESTED** [review]

**Review by etsauer — APPROVED** [review]

**Pull request merged**